src/node_buffer.cc

```diff
@@ -197,7 +298,7 @@ Local<Object> New(Environment* env,
 }
 
 
-Local<Object> Use(Isolate* isolate, char* data, uint32_t length) {
+Local<Object> Use(Isolate* isolate, char* data, size_t length) {
```

Comment:
Since it'll have to return a `MaybeLocal<>` changing the argument type wouldn't matter. Though since it's still confined to the uint32 space might as well change it back. Made the switch because size_t is what ArrayBuffer::New takes.